content/applications/general/email_communication/email_servers.rst

```diff
@@ -252,3 +166,68 @@ Now, go to :menuselection:`Email Marketing --> Settings` and enable :guilabel:`D
 With these settings, Odoo uses the server with the lower priority for transactional emails, and the
 server here selected for mass mails. Note that in this case, the domain's Sender Policy Framework
 (SPF) records must be set to include both transactional and mass mail servers.
+
+.. _email_communication/inbound_messages:
+
+Manage inbound messages
+=======================
+
+Odoo relies on generic email aliases to fetch incoming messages.
+
+* **Reply messages** of messages sent from Odoo are routed to their original discussion thread (and
+  to the inbox of all its followers) by the catchall alias (**catchall@**).
+* **Bounced messages** are routed to **bounce@** in order to track them in Odoo. This is especially
+  used in `Odoo Email Marketing <https://www.odoo.com/page/email-marketing>`__ to opt-out invalid
+  recipients.
```

Comment:
No no,
The bounce address is used as a Return-Path (see: https://sendgrid.com/blog/what-is-return-path/) 

This is adding the red envelop on the record that has failed.
Big improvement is coming in master to be able to correctly parse the bounce reason instead of the current&ugly `No Error`

On the mass mailing side, the bounces are only opt-out based on this:
        

> If it bounced too much times (5)*
> in the last month and the bounced are at least separated by one week,
> to avoid blacklist someone because of a temporary mail server error,
> then the email is considered as invalid and is blacklisted.

\* : See _routing_handle_bounce in odoo/addons/mass_mailing/models/mail_thread.py

A log note is added on the contact under `Blacklisted Email Addresses` on the Email Marketing Configuration Menu

Reply:
This is super helpful information. I've added this to the doc. Thanks for sharing 💯